src/js/views/globals/search/patient-search_views.js

```diff
@@ -42,24 +65,11 @@ const PatientSearchPicklist = Picklist.extend({
   itemTemplate: hbs`
     {{matchText text search}}{{~ remove_whitespace ~}}
     <span class="patient-search__picklist-item-meta">{{formatDateTime birth_date "MM/DD/YYYY"}}</span>
-    {{#each identifiers}}<span class="patient-search__picklist-item-meta">{{this}}</span>{{/each}}
   `,
   itemTemplateContext() {
-    const settings = this.state.get('settings');
-    const settingsIdentifiers = get(settings, 'result_identifiers');
-
-    const patientIdentifiers = this.model.get('identifiers');
-
-    const identifiers = map(settingsIdentifiers, settingsIdentifier => {
-      const identifierToShow = find(patientIdentifiers, { type: settingsIdentifier });
-
-      return identifierToShow && identifierToShow.value;
-    });
-
```

Comment:
Removed code that handles showing identifiers in the search results. Will be handled in a future story.

Also removed those portions of the Cypress tests.